ansible/roles/rhacm_add_clusterimagesets/tasks/main.yml

```diff
@@ -20,6 +20,9 @@
     - name: openshift-4.8.0-fc.2
       releaseImage: quay.io/openshift-release-dev/ocp-release:4.8.0-fc.2-x86_64
       dest: "{{ rhacm_install_directory }}/rhacm_cis/openshift-4.8.0-fc.2-cis.yaml"
+    - name: openshift-4.8.0-fc.3
```

Comment:
Should we variabilize this entire section?

Reply:
Yeah since this will continuously change, probably best to just migrate the config into the vars file

Reply:
Migrated this to the vars file.